nixos/tests/ipv6.nix

```diff
@@ -11,6 +11,7 @@ import ./make-test-python.nix ({ pkgs, lib, ...} : {
     # Remove the interface configuration provided by makeTest so that the
     # interfaces are all configured implicitly
     { client = { ... }: { networking.interfaces = lib.mkForce {}; };
+      client_no_privacy_extensions = { ... }: { networking.interfaces = lib.mkForce {}; networking.ipv6PrivacyExtensions = "disabled"; };
```

Comment:
This appears to be unused... am I missing something? Also, can you break this line?